README.md

```diff
@@ -1103,6 +1103,24 @@ sudo lunatik run examples/filter/sni false   # runs the Lua kernel script
 sudo xdp-loader load -m skb <ifname> https.o # loads the XDP/eBPF program
 ```
 
+### dnsblock
+
+[dnsblock](examples/dnsblock) is a kernel extension composed by
+using the netfilter kernel facility and a Lua kernel script to filter DNS packets. 
+This kernel extension drops any outbound DNS packet with questions matching the blacklist
+provided by the user.
+
+#### Usage
+
+```
+sudo make examples_install						# installs examples
+cd examples/dnsblock
+make											# builds the userspace extension for netfilter
+sudo make install								# installs the extension to Xtables directory
```

Comment:
it's unaligned.. please, check how it's rendered on [GitHub](https://github.com/sheharyaar/lunatik/tree/xtables-rcu?tab=readme-ov-file#usage-8)..